docs/.vitepress/vitepress/components/navbar/vp-translation.vue

```diff
@@ -23,9 +23,15 @@ const toTranslation = () => {
               v-for="l in langs"
               :key="l"
               :class="{ language: true, selected: l === lang }"
-              @click="switchLang(l)"
             >
-              {{ languageMap[l] }}
+              <a
+                :href="`/${l}${
+                  router.route.path.replace(/^\/[^\/]+/, '') || '/'
```

Comment:
This calculation logic is more suitable for integration with the implementation of switchLang.